### ISSUE TITLE
Feature Request: Multi-Line-Input and generic tree

### ISSUE THREAD

**Issue by KevinW1998** [IgorTimofeev/GUI#11]
I really love the GUI API here, but I somewhat miss two GUI components:
* Multi-Line input field
* Generic Tree (as it seems there is only filesystem-tree)

**Comment by IteratorW** [comment]
Well, it should be an author's comment, but why wouldn't you add your own objects and even make a pull request with 'em?

**Comment by IgorTimofeev** [comment]
@KevinW1998 Yes, multi-line input will be added someday, although this will lead to a fairly large-scale redeveloping of existing objects like codeView.

There's already a GUI.tree object, GUI.filesystemTree just extends it. Tree allows to add any string data to it, here is example for RAM stacktrace:

![2019-01-08 2 07 43](https://user-images.githubusercontent.com/11760002/50798665-37d71300-12ea-11e9-9eeb-09f264306ef1.png)

Unfortunately, there's tons of undocumented features like this, i'm too lazy to write some info about them. Need to focus on MineOS Standalone version and all its documentation
